Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM metacpan/metacpan-base:latest
+ADD . /code
+WORKDIR /code
+RUN cpm install -g --cpanfile cpanfile
```

Comment:
You may want to add `--without-test` to the `cpm` install, as the image doesn't need to run the tests for the dependencies nor include the test modules.